docs/templates/applications.md

```diff
@@ -206,6 +206,7 @@ The default input size for this model is 299x299.
     It should have exactly 3 inputs channels,
     and width and height should be no smaller than 71.
     E.g. `(150, 150, 3)` would be one valid value.
+- nb_classes: optional number of categories to classify images into, only to be specified if `include_top` is True
```

Comment:
- please rename to `classes` everwhere
- add: `and if no `weights` argument is specified.`.
- break the doc line to respect line length limitations.